src/node/rpc/member_frontend.h

```diff
@@ -330,6 +382,162 @@ namespace ccf
       return tx_modules->remove(name);
     }
 
+    void remove_jwt_keys(kv::Tx& tx, std::string issuer)
+    {
+      auto keys = tx.get_view(this->network.jwt_public_signing_keys);
+      auto key_issuer =
+        tx.get_view(this->network.jwt_public_signing_key_issuer);
+
+      key_issuer->foreach(
+        [&issuer, &keys, &key_issuer](const auto& k, const auto& v) {
+          if (v == issuer)
+          {
+            keys->remove(k);
+            key_issuer->remove(k);
+          }
+          return true;
+        });
+    }
+
+    bool set_jwt_public_signing_keys(
```

Comment:
Proposals so far were fairly simple and only had a few reasons to fail. However, this one is probably the most complex one we've had so far. Proposal execution currently suffers from https://github.com/microsoft/CCF/issues/1031, which means for example that even if the proposal fails, what has been written to a view will be written to the KV anyway. 

This is bad and should be fixed. I think this PR makes me think we should fix this sooner than later.

Reply:
I worked around this for now by doing all kv writes at the end.